src/components/Terminal/PTerminal.vue

```diff
@@ -10,10 +10,15 @@
     <div class="p-terminal__code">
       <template v-for="(line, index) in commands" :key="index">
         <p class="p-terminal__code-line">
-          {{ line }}
+          $ {{ line }}
```

Comment:
Instead of a hard-coded $, I'd prefer we went with a prefix string property, defaulting to empty:

```suggestion
          <span v-if="prefix">{{ prefix }} </span> {{ line }}
          
          ...
          
          
          const props = defineProps<{
            ...
            prefix?: string  
          }>()
```

Reply:
We probably want all terminal components to have the same prefix across all our use cases though right? I feel like having this be a prop (or at least not having a default) will mean this will get copy/paste all over and likely to be missed when not pasting. 

Also, what's an example of a different prefix that might be used?

Reply:
Maybe - as you mentioned above, sometimes we use these for more display purposes and other times we use it to spell out a command or series of commands a user should run. The former probably doesn't require a prefix and the latter probably does. We're pretty bash-heavy in examples but having some flexibility here leaves the door open for any sort of prompt. Using the example from above might look something like:

```
$ ls
docs src
$ cd docs
/docs $ printenv
...
```

or the same thing in powershell:

```
PS C:\> ls
docs src
PS C:\> cd C:\docs
PS C:\docs> Get-ChildItem env:
...
```

Reply:
writing that powershell example was actually painful, maybe we shouldn't do this after all so we don't encourage powershell usage